src/worker.js

```diff
@@ -274,3 +264,57 @@ this.onmessage = function(e) {
     throw e;
   }
 };
+
+#if ENVIRONMENT_MAY_BE_NODE
+// Node.js support
+if (typeof process === 'object' && typeof process.versions === 'object' && typeof process.versions.node === 'string') {
+  // Create as web-worker-like an environment as we can.
+  self = {
+    location: {
+      href: __filename
+    }
+  };
+
+  var onmessage = this.onmessage;
+
+  var nodeWorkerThreads = require('worker_threads');
+
+  Worker = nodeWorkerThreads.Worker;
+
+  var parentPort = nodeWorkerThreads.parentPort;
+
+  parentPort.ref(); // TODO: investigate when we need this
+
+  parentPort.on('message', function(data) {
+    onmessage({ data: data });
+  });
+
+  var nodeFS = require('fs');
+
+  var nodeRead = function(filename) {
+    return nodeFS.readFileSync(filename).toString();
```

Comment:
```suggestion
    return nodeFS.readFileSync(filename, 'utf8');
```

(Just a minor thing, it avoids creating an intermediate `Buffer`/copying it into a string)

Reply:
Nice, thanks! I think it's best to open a separate PR for that later as I see we have more than one existing call like that that could be improved that way.